packages/kit/src/index.css

```
@@ -585,6 +585,33 @@
+
+  .contained-action-card__content {
+    @apply flex min-w-0 flex-1 basis-64 items-center gap-2;
+  }
+
+  .contained-action-card__content > svg {
+    @apply shrink-0;
+  }
+
+  .contained-action-card__text {
+    @apply min-w-0 flex-1;
+    overflow-wrap: anywhere;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)